sgtcpsocket.cpp

```diff
@@ -176,19 +179,21 @@ void SGTCPSocket::read_socket_send_signal()
             success_flag = true;
             message_ba.remove(0, 4);
             success_message = message_ba;
-            qDebug() << "Server message: " << success_message;
+            //qDebug() << "Server message: " << success_message;
         }
         else if (QString(message_ba).left(4) == "FAIL")
         {
             // Set the fail flag
             fail_flag = true;
             message_ba.remove(0, 4);
             // Display the failure message to user
-            QMessageBox timeout_box;
-            timeout_box.setText("Error");
-            timeout_box.setInformativeText(message_ba);
-            timeout_box.setIcon(QMessageBox::Warning);
-            timeout_box.exec();
+            if (message_ba.size() > 0) {
```

Comment:
Should have already been like this... but now the error box only shows up if the FAIL message from the server actually has a message after it